core/app/assets/js/ui/card/card.js

```diff
@@ -17,3 +17,7 @@ Card.propTypes = {
 Card.defaultProps = {
 	className: '',
 };
+
+Card.displayName = 'Card';
+
+export default Card;
```

Comment:
Why do you need these?

Reply:
When the functional component is not using the keyword `function` to declare the function (like this case), `eslint` requires the property `displayName` on the function.
We cannot use the keyword `function` for declaration because of `React.forwardRef()` that wraps our component to create another component that can accept a `ref` as a prop.

Reply:
@kobizz